pkg/cable/libreswan/libreswan.go

```diff
@@ -173,12 +179,12 @@ func (i *libreswan) Init() error {
 }
 
 // Line format:
-// 006 #3: "submariner-cable-cluster3-172-17-0-8-0-0", type=ESP, add_time=1590508783, inBytes=0, outBytes=0, id='172.17.0.8'
+// 006 #3: "submariner-cable-cluster3-172-17-0-8-v4-0-0", type=ESP, add_time=1590508783, inBytes=0, outBytes=0, id='172.17.0.8'
 // or:
-// 006 #2: "submariner-cable-cluster3-172-17-0-8-0-0"[1] 3.139.75.179, type=ESP, add_time=1617195756, inBytes=0, outBytes=0, \
+// 006 #2: "submariner-cable-cluster3-172-17-0-8-v4-0-0"[1] 3.139.75.179, type=ESP, add_time=1617195756, inBytes=0, outBytes=0, \
 // id='@10.0.63.203-0-0'"
 // .
-var trafficStatusRE = regexp.MustCompile(`.* "([^"]+)"[^,]*, .*inBytes=(\d+), outBytes=(\d+).*`)
+var trafficStatusRE = regexp.MustCompile(`.* "([^"]+-v[46]-[0-1]-[0-1])"[^,]*, .*inBytes=(\d+), outBytes=(\d+).*`)
```

Comment:
What about migration, ie do we need to handle parsing connections with the prior format obtained from `whack`? (it seems not...)

Reply:
Good point, 
I think it should be fine. 

IIUC, [Cablenegine](https://github.com/submariner-io/submariner/blob/devel/pkg/cableengine/cableengine.go#L183) will not find an existing active connection even though it exists (with previous naming convention), it needs to call libreswan to Connect to Endpoint and I think libreswan should apply the same connection with the new connection name.